content/troubleshooting/docs/reverse-proxy-sparkui.md

```diff
@@ -0,0 +1,290 @@
+# **Connect to SparkUI without Port Fowarding**
+
+This is an example of connecting to SparkUI running on Spark's driver pod via a reserve proxy solution, without an access to the kubectl tool or AWS console. We demostrate how to set it up via three EMR on EKS deployment methods: Spark Operator, JobRun API, and spark-submit.
+
+## Launch EMR on EKS jobs via Spark Operator
+
+There are 3 Steps to set up in EKS:
+
+1.Create a SparkUI reverse proxy and an ALB first. 
+
+The sample yaml file is in the [Appendix](#deploymentyaml). Make sure the EMR on EKS namespace at line #25 is updated by your namespace:
```

Comment:
I think you meant line 171 in this file. Maybe we could explicitly call it out.

Reply:
the sample yaml file doesn't have line 171. it refers to the #25 in deployment.yaml file.